src/lib/components/namespace-list.svelte

```diff
@@ -1,19 +1,79 @@
 <script lang="ts">
   import { page } from '$app/stores';
+  import { createEventDispatcher } from 'svelte';
+  import { onMount } from 'svelte';
 
   import Icon from '$holocene/icon/icon.svelte';
-
-  import { onMount } from 'svelte';
   import EmptyState from '$lib/holocene/empty-state.svelte';
-  import { createEventDispatcher } from 'svelte';
+  import Loading from '$lib/holocene/loading.svelte';
+
+  import { lastUsedNamespace } from '$lib/stores/namespaces';
+  import { workflowSorts, workflowFilters } from '$lib/stores/filters';
+  import { goto } from '$app/navigation';
+  import {
+    routeForWorkflows,
+    routeForSchedules,
+  } from '$lib/utilities/route-for';
+  import { fetchNamespaces } from '$lib/services/namespaces-service';
 
-  export let getNamespaceList: () => Promise<NamespaceItem[]> = null;
```

Comment:
We'll need to rethink this a bit since cloud relies on passing in GetNamespaceList to FullNav and passing this down to namespace-list.

Reply:
I've added back `getNamespaceList` and the other props being passed to `Navigation` ✅ 

One thing I noticed is that we are currently passing `$lastUsedNamespace` (a `string`) to `activeNamespace` (a `Namespace`) in cloud currently which seems incorrect 🤔